src/player.js

```diff
@@ -574,7 +574,8 @@ export default class Player extends FakeEventTarget {
     if (this._reset) return;
     this.pause();
     //make sure all services are reset before engine and engine attributes are reset
-    this._externalCaptionsHandler.reset();
+    // $FlowFixMe
+    this._externalCaptionsHandler.reset(this._tracks.filter(track => track.external));
```

Comment:
what is the issue with the typing?

Reply:
the external property doesn't exist on Track class sp flow doesn't like it but filter using `undefined` same as `false` so there is no problem.